tests/validation/tests/v3_api/test_istio.py

```diff
@@ -70,6 +103,19 @@ def test_istio_resources():
     create_and_test_bookinfo_routing(app_client, app_ns, gateway_url)
 
 
+def test_istio_custom_answers(enable_all_options):
```

Comment:
are these the same deployments if we run this test with an older version of istio? Is there any other impact on running these tests w/ an older version?

Reply:
I added a `skipif` tag to these now for previous versions. Good catch!